packages/ckeditor5-dev-env/lib/release-tools/tasks/releasesubrepositories.js

```diff
@@ -208,28 +220,79 @@ module.exports = function releaseSubRepositories( options ) {
 		} );
 	}
 
-	function releasePackages( releaseOptions ) {
+	function pullAndPushPackages() {
+		return executeOnPackages( pathsCollection.packages, repositoryPath => {
+			process.chdir( repositoryPath );
+
+			const packageJson = getPackageJson( repositoryPath );
+			log.info( `Synchronizing a local repository with the remote for "${ packageJson.name }"...` );
+
+			// Push updated dependencies and changelog if was generated.
+			exec( 'git pull && git push' );
```

Comment:
We can't pull after tagging a new version. That would break the tag.

Reply:
Sure. Pull should be done before tagging.

Reply:
Those commands should not try to pull anything. The assumption is that no one will do any changes during the process. In other words – let's assume that the releaser pulled everything, (optionally) validate that there are no new changes before committing anything, and then just push.

Reply:
I removed pulling and didn't introduce your optionally validation step.

Let's assume that everything is pulled before starting release.